lansa-vmss-windows-autoscale-sql-database/Scripts/Deploy-AzureResourceGroup.ps1

```diff
@@ -1,6 +1,6 @@
 ﻿#Requires -Version 3.0
-#Requires -Module AzureRM.Resources
-#Requires -Module Azure.Storage
+#Requires -Module Az.Resources
```

Comment:
how is this used?

Reply:
The script was provided as part of the original template from Microsoft. Generally its not used. It may be used to deploy a stack from the command line.

Reply:
I use it in development.